tests/unit/db/vertica/test_vertica_db_driver.py

```diff
@@ -14,13 +14,12 @@ def test_unload(self):
 
         self.mock_records_unload_plan.records_format.hints = vertica_format_hints
         self.mock_directory.scheme = 's3'
-        export_count = self.vertica_db_driver\
-            .unloader().unload(schema='myschema',
-                               table='mytable',
-                               unload_plan=self.mock_records_unload_plan,
-                               directory=self.mock_directory)
-
-        self.assertEqual(579, export_count)
+        with self.assertRaises(NotImplementedError):
```

Comment:
how come this is back to `NotImplementedError` and not `AttributeError` like the other cases?